index.js

```diff
@@ -20,25 +18,46 @@ function get_new_question() {
         let quote_element = document.getElementById('quote')
         quote_element.textContent = current_movie.quote
         let answer_element = document.getElementById('answer')
+        answer_element.dataset.year = current_movie.year
+        answer_element.dataset.title = current_movie.title
         answer_element.style.display = 'none'
         answer_element.innerHTML = '<h2>' + current_movie.title + '</h2>'
```

Comment:
Is this still needed now that you're storing the answer in the dataset? Seems like you're now keeping track of the answer in two places.

Reply:
Yes we believe we do